Makefile

```diff
@@ -2,6 +2,7 @@ ISA ?= rv64imafdc
 ABI ?= lp64d
 TARGET_BOARD := U74
 BOARD_FLAGS	:=
+HWBOARD ?= starlight-a1
```

Comment:
VisionFive is recommended as the default since it is about to be mass-produced and sold.